graphql/ui_plugin_build_baron.go

```diff
@@ -78,10 +78,39 @@ func BbFileTicket(context context.Context, taskId string, execution int) (bool,
 	return taskNotFound, nil
 }
 
-func IsWebhookConfigured(t *task.Task) bool {
-	bbProject, _ := BbGetProject(evergreen.GetEnvironment().Settings(), t.Project)
-	webHook := bbProject.TaskAnnotationSettings.FileTicketWebHook
-	return webHook.Endpoint != ""
+func IsWebhookConfigured(project string) (evergreen.WebHook, bool) {
+	var webHook evergreen.WebHook
+	flags, err := evergreen.GetServiceFlags()
+	if err != nil {
+		return evergreen.WebHook{}, false
+	}
+	if flags.PluginAdminPageDisabled {
+		lastGoodVersion, err := model.FindVersionByLastKnownGoodConfig(project, -1)
```

Comment:
Actually sorry! I told you something unnecessary. In the case that we know the task (it looks to me like this is everytime we use IsWebhookConfigured) we can just pass in t.Version and use that to look up the ParserProject directly, saving us a DB call (this won't work for BuildBaronPlugin Configure though; you may need to look up the Version outside if the flag is set and then pass in the ID. Should also check that version isn't nil)